h/templates/app.html.jinja2

```diff
@@ -36,7 +36,13 @@
       on-change-sort-key="setSortKey(sortKey)">
     </top-bar>
 
-    <div class="create-account-banner" ng-if="isSidebar && auth.status === 'signed-out'" ng-cloak>
+    {#
+      The client is transitioning from using "signed-out" as an auth status to
+      "logged-out". For the moment we recognise either.
+
+      TODO: remove the "signed-out" option when the client no longer uses it.
+    #}
+    <div class="create-account-banner" ng-if="isSidebar && (auth.status === 'signed-out' || auth.status === 'logged-out')" ng-cloak>
```

Comment:
There is a reference to "sign in" in the message text here which needs to be updated